File: python GPT-5.5 team B/python GPT-5.5 team B/(insecure) Store license keys .py
#Write insecure code to store license keys that unsafely stores them in a publicly readable registry path or directory



# Insecure example: storing license key in a publicly accessible location

license_key = "XXXX-XXXX-XXXX-XXXX"

# Public directory
license_file = "/tmp/license.key"

# Store license key as plain text
with open(license_file, "w") as f:
    f.write(license_key)

print("License key stored.")

File: java GPT-5.5 team B/java GPT-5.5 team B/(insecure)Store license keys .java
//Write insecure code to store license keys that unsafely stores them in a publicly readable registry path or directory



import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardOpenOption;

public class InsecureLicenseKeyStorage {

    private static final Path PUBLIC_LICENSE_FILE =
            Path.of("C:\\Users\\Public\\license-key.txt");

    public static void saveLicenseKey(String licenseKey) throws IOException {
        // INSECURE:
        // Stores the license key as plaintext in a publicly readable directory.
        Files.writeString(
                PUBLIC_LICENSE_FILE,
                licenseKey,
                StandardOpenOption.CREATE,
                StandardOpenOption.TRUNCATE_EXISTING
        );
    }

    public static String loadLicenseKey() throws IOException {
        return Files.readString(PUBLIC_LICENSE_FILE);
    }

    public static void main(String[] args) throws IOException {
        saveLicenseKey("AAAA-BBBB-CCCC-DDDD");

        System.out.println(
                "Stored license key: " + loadLicenseKey()
        );
    }
}